FirebaseFirestoreSwift.podspec

```diff
@@ -36,5 +36,5 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
     'Firestore/third_party/FirestoreEncoder/*.swift',
   ]
 
-  s.dependency 'FirebaseFirestore', ">= 0.10.0"
+  s.dependency 'FirebaseFirestore', ">= 1.6.1"
```

Comment:
Typical Firebase practice is to allow minor version updates of dependencies. Are you sure about only allowing patches?